include/FindCircle.hpp

```diff
@@ -21,7 +21,7 @@ struct Circle_st {
 /// gradhist that is built in this function.  Set any gradient values
 /// that are below cut_below * maximum_gradient to zero, then fit to
 /// circle. Returns Circle_st representing the best fit circle.
-Circle_st find_circle_max_grad( const TH2D* in, TH2D*& grad, double cut_below = 0.8 );
+Circle_st find_circle_max_grad( const TH2D* in, TH2D*& grad, double cut_below = 0.5 );
```

Comment:
Non-accelerometer related? Should include such changes in a different PR, or include in the title/description of this one.

Reply:
Probably just a mistake, I will make these corrections !